_posts/2016-1-25-Blog-Building.md

```diff
@@ -0,0 +1,158 @@
+		</td>
+	</tr>
+
+	<tr>
+		<td>
+			<a href="http://fritzing.org/home/" target="\_blank">fritzing</a>
+		</td>
+
+		<td>
+			<center><a href="http://fritzing.org/home/" target="\_blank"><img src="/images/Logos/fritzing_logo.png" alt="Fritzing Logo"></img></a></center>
+		</td>
+
+		<td>
+			Fritzing is a very cool open source program for designing circuit schematics and diagrams. I use it to show clean representations of our Arduino projects for the blog. You can also submit your design to then get printed by fritzing, but that's a paid service.
+		</td>
+	</tr>
+
+	<tr>
+		<td>
+			<a href="http://chrispederick.com/work/web-developer/" target="\_blank">Web Developer Browser Plug-In</a>
+		</td>
+
+		<td>
+			<center><a href="http://chrispederick.com/work/web-developer/" target="\_blank"><i class="fa fa-cog fa-5x" style="color: gray;"></i></a></center>
+		</td>
+
+		<td>
+			This plugin, available on Chrome, Firefox, and Opera, is super useful for testing. I use it to test how the blog will look for people with color-blindness, people who have CSS disabled, and people who may be reading the blog with only screen readers.
+		</td>
+	</tr>
+
+	<tr>
+		<td>
+			<a href="http://fortawesome.github.io/Font-Awesome/" target="\_blank">Font Awesome</a>
+		</td>
+
+		<td>
+			<center><a href="http://fortawesome.github.io/Font-Awesome/" target="\_blank"><img src="/images/Logos/font-awesome_logo.png" alt="Font Awesome Logo"></img></a></center>
+		</td>
+
+		<td>
+			Font Awesome is a free font of logos and icons. I use it on the blog to create the buttons at the bottom of the page to link to an RSS feed and my Github and LinkedIn accounts. I used in the line above to create the gear icon!
+		</td>
+	</tr>
+
+	<tr>
+		<th colspan="3">Techniques</th>
+	</tr>
+
+	<tr>
+		<td>
+			Run the blog locally before posting
+		</td>
+
+		<td>
+			<center><img src="/images/Logos/cmd_logo.png" alt="Command Prompt Logo"></img></center>
+		</td>
+
+		<td>
+			It's very helpful to see what your blog will look like as you're building it and before you show it to everyone else. To do this, I use my Command Prompt (just search of it on your computer) to navigate into my blog directory and then serve it on my local computer. The commands are super basic:
+			<script src="https://gist.github.com/madjenjen/f23aa17199c2fcf3c0ec.js"></script>
+			Then you'll use your web browser to go to <a href="http://127.0.0.1:4000" target=\_blank">http://127.0.0.1:4000</a> to check out your blog!
+
+			Fun fact: Jekyll is smart enough to know the date you want your blog posted, and it won't show it even locally if you put a future date on it, so for testing purposes, title your blog with today's date and then change it when you're ready to push it to the world.
```

Comment:
Installing Jekyll is far from trivial. You should either mention that getting Jekyll installed is covered in the article mentioned at the top of the post, link to the Jekyll install docs, or both. Otherwise the commands in the gist above won't work out-of-the-box.